packages/wdio-logger/src/node.js

```diff
@@ -126,20 +126,36 @@ export default function getLogger (name) {
 }
 
 getLogger.setLevel = (name, level) => loggers[name].setLevel(level)
-getLogger.setLogLevelsConfig = (logLevels = {}) => {
+getLogger.setLogLevelsConfig = (logLevels = {}, wdioLogLevel = DEFAULT_LEVEL) => {
+    /**
+     * set log level
+     */
+    if (process.env.WDIO_LOG_LEVEL === undefined) {
+        process.env.WDIO_LOG_LEVEL = wdioLogLevel
+    }
+
     logLevelsConfig = {}
 
-    Object.keys(logLevels).forEach(loggerName => {
-        const logLevelName = getLogLevelName(loggerName)
-        logLevelsConfig[logLevelName] = logLevels[loggerName]
+    /**
+     * build logLevelsConfig object
+     */
+    Object.entries(logLevels).forEach(([logName, logLevel]) => {
+        const logLevelName = getLogLevelName(logName)
+        logLevelsConfig[logLevelName] = logLevel
     })
 
-    Object.keys(loggers).forEach(loggerName => {
-        const logLevelName = getLogLevelName(loggerName)
+    /**
+     * set log level for each logger
+     */
+    Object.keys(loggers).forEach(logName => {
+        const logLevelName = getLogLevelName(logName)
 
-        if (logLevelsConfig[logLevelName] !== undefined) {
-            loggers[loggerName].setLevel(logLevelsConfig[logLevelName])
-        }
+        /**
+         * either apply log level from logLevels object or use global logLevel
+         */
+        const logLevel = logLevelsConfig[logLevelName] !== undefined ? logLevelsConfig[logLevelName] : process.env.WDIO_LOG_LEVEL
```

Comment:
```suggestion
        const logLevel = logLevelsConfig[logLevelName] || process.env.WDIO_LOG_LEVEL
```

Reply:
@christian-bromann I'm afraid about case when `logLevelsConfig[logLevelName] === 0`.
In such case this ternary won't work 
`logLevelsConfig[logLevelName] || process.env.WDIO_LOG_LEVEL`

Reply:
yeah true, even tho it seems unlikely, how about doing

```
typeof logLevelsConfig[logLevelName] !== 'undefined'
```

I am kind of careful about comparing variables like this, e.g. `[undefined] == '' // true`

Reply:
I'll change to `typeof logLevelsConfig[logLevelName] !== 'undefined'` in the evening